src/Configuration/Option.php

```diff
@@ -111,6 +113,11 @@ final class Option
      */
     public const CACHE_DIR = 'cache_dir';
 
+    /**
+     * @var class-string<CacheStorage>
+     */
+    public const CACHE_CLASS = FileCacheStorage::class;
```

Comment:
new config option to define which caching-backend should be used.

for the rector-end-user I guess this is mostly irrelevant, but we can use it in github actions since these always start from scratch and the file-cache is not persisted between runs.

in-memory is waaaaay faster then the file impl.

Reply:
Thanks :+1: 

Could you 1-2 sentense comment from this to this constant? Just to people know they should not use it.
Maybe mark with `@internal`